src/middlewares/request.middleware.js

```diff
@@ -295,10 +295,26 @@ function checkChannelID (req, res, next) {
   next()
 }
 
+function addChannelFilters (req, res, next) {
+  // Assuming that the request will not send the filter of channels
+  filterService.getChannelSearchString(req, function (err, channels) {
```

Comment:
why do we need call filterservice on every API call? i guess the configuration can not get updated dynamically, and config change requires a server restart. the searchString can actually be cached on server start.